docs/architecture.md

```diff
@@ -57,7 +57,7 @@ To facilitate this, the `PulpContext` provides the `needs_plugin` and `has_plugi
 ```python
 class MyEntityContext(PulpEntityContext):
     def show(self, href):
-        if not self.pulp_ctx.has_plugin("pulp_my_content", min_version="1.2.3"):
+        if not self.pulp_ctx.has_plugin(PluginRequirement("pulp_my_content", min="1.2.3")):
```

Comment:
AFAICT, these changes are not related to the inverted feature that you're also introducing here. Is that correct?

Reply:
In a way. Those functions have the "old" interface aka "passing parameters individually" and the new one, "passing the `PluginRequrement` as a single object".
The old interface was meant to go away anyway, and i didn't bother to add the `inverted` to it. So i decided for removing it here.
Why do we have the sligtly more complex "new" interface? `needs_plugin` works deferred, and it was very convenient to store and pass around the `PluginRequirement` objects.